demo/docker/docker-compose.yml

```diff
@@ -43,6 +43,19 @@ services:
     depends_on:
       - zookeeper
 
+  kafka-manager:
```

Comment:
Isn't it too much for demo? Soon you'll need MacPro to run it - event now it's really memory hog because of elastic etc....

Reply:
Take it easy :) I think it's a good example how to configure fullstack for project.